infra/modules/notifications-app/resources/main.tf

```diff
@@ -0,0 +1,10 @@
+resource "aws_pinpoint_app" "app" {
+  name = var.name
+}
+
+resource "aws_pinpoint_email_channel" "app" {
+  application_id    = aws_pinpoint_app.app.application_id
+  configuration_set = var.email_identity_config
```

Comment:
see my other comment, i think we can get rid of this option and also get rid of the var.email_identity_config variable